### PR TITLE
Split patches per release

### PR DESCRIPTION
**Pull request by wmousa**
Create a release directory and its own patches

### CODE REVIEW

**Comment by wmousa** [comment]
@moshe010

File: openstack/README.md [hunk 1]
```diff
@@ -2,15 +2,17 @@
 
 ## Apply patches
 
-After preparing the setup and creating the ovs-forwarder container, apply the following patches:  
+After preparing the setup and creating the ovs-forwarder container, apply the following patches of related release:
```
Comment (adrianchiris) [inline openstack/README.md]: > of related release

for the relevant OpenStack release

File: openstack/README.md [hunk 2]
```diff
@@ -2,15 +2,17 @@
 
 ## Apply patches
```
Comment (adrianchiris) [inline openstack/README.md:15]: Please describe in high level what components need to be patched and what each does (at high level)

File: openstack/ussuri/ovs-kernel/os-vif.patch [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+ 
+ LOG = logging.getLogger(__name__)
+ 
+@@ -62,6 +63,21 @@ def _update_device_mtu(dev, mtu):
+ 
+ 
+ @privsep.vif_plug.entrypoint
++def create_vdpa_port(vif_name, pf_pci, vf_pci, vf_num, vdpa_socket_path):
++    container_ovs = containerovsdb_lib.BaseOVS(connection='tcp:127.0.0.1:6000')
++    container_ovs.create_ovs_vif_port("br0-ovs", vif_name,
```
Comment (adrianchiris) [inline openstack/ussuri/ovs-kernel/os-vif.patch:33]: Feels like these parameters should really go into os-vif config opts as they depend on how the container was built/started, but i guess thats how it was before this PR.

so something to consider as improvment.

**Comment by adrianchiris** [comment]
generally looks ok, some small comments

File: openstack/README.md (hunk 2 above)
Comment (moshe010) [reply openstack/README.md:15]: @wmousa, I don't see you address this comment

**Pull request merged**